PyInstaller/compat.py

```diff
@@ -869,20 +872,21 @@ def check_requirements():
         if 'win32api' in sys.modules or 'pywintypes' in sys.modules:
             # Users should never see this error; if it occurs, it means someone
             # wasn't careful and added an import where it shouldn't be
-            raise SystemExit("Internal error: early pywin32 import was introduced")
-        
+            logger.warning(
+                "Internal error: early pywin32 import was introduced")
+            return
```

Comment:
That's because the test suite wouldn't run with the pywin32 problems. It will be rebased after that PR is merged.